src/components/BatchSearchFilterDropdown.vue

```diff
@@ -1,7 +1,13 @@
 <template>
   <batch-search-filter class="batch-search-filter-dropdown" :id="id" :name="name" :active="isActive">
     <keep-alive>
-      <selectable-dropdown v-model="selectedValues" :items="items" deactivate-keys :multiple="multiple" class="shadow-none border-0">
+      <selectable-dropdown
+        v-model="selectedValues"
```

Comment:
I think we can add the vue guidelines to have order in the props (enhance readability from my POV):
- v-if
- bindings
- alphabetical order

Reply:
Ah yes,  that'd be nice! do you know where I can find these as eslint rules?

Reply:
It's from the plugin eslint vue https://eslint.vuejs.org/
https://eslint.vuejs.org/rules/attributes-order.html
my bad I think it's already enforced...